lib/broker.go

```diff
@@ -0,0 +1,142 @@
+package sebak
+
+import (
+	"io"
+)
+
+type EventType string
+
+const (
+	EventDummy  EventType = "dummy"
+	EventDummy2           = "dummy2"
+)
+
+type EventBroker struct {
+	brokers map[EventType]*Broker
+}
+
+func NewEventBroker() *EventBroker {
+	e := &EventBroker{
+		brokers: map[EventType]*Broker{},
+	}
+	e.brokers[EventDummy] = NewBroker()
```

Comment:
IMHO, This statement `e.brokers[EventDummy] = ..` is not necessary.

Reply:
AddBroker much better 👍